src/components/StablecoinsTable.tsx

```diff
@@ -47,23 +47,15 @@ const StablecoinsTable = ({
   }
 
   return (
-    <Table
-      variant="unstyled"
-      my={8}
-      borderRadius="sm"
-      border={`1px solid ${textColor}`}
-      bg="background.base"
-      mb={8}
-      minW="720px"
-    >
-      <Thead bg="ednBackground" color="text200">
-        <Tr borderBottom={`1px solid ${textColor}`} mb="1px">
+    <Table variant="unstyled" my={8} bg="background.base" mb={8} minW="720px">
+      <Thead bg="background.highlight" color="body.medium">
+        <Tr>
           {columns.map((column, idx) => (
             <Th
               key={idx}
-              py={5}
+              fontWeight="bold"
               fontSize="md"
-              fontWeight="normal"
+              verticalAlign="inherit"
```

Comment:
Do we need this?
```suggestion
```

Reply:
Trust me, it's absolutely necessary. Without it, the text won't be centered anymore.

Reply:
![image](https://github.com/ethereum/ethereum-org-website/assets/57232813/5330d975-3d32-4993-9ad0-21bbd5f35749)

I've made a diagram where you can clearly see the difference between having "verticalAlign="inherit"" and not having it.

cc @wackerow